cmd/root/root.go

```diff
@@ -97,7 +98,28 @@ func flagErrorFunc(c *cobra.Command, err error) error {
 // Execute adds all child commands to the root command and sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute(ctx context.Context, cmd *cobra.Command) error {
-	// TODO: deferred panic recovery
+	defer func() {
+		r := recover()
+
+		// No panic. Return normally.
```

Comment:
nit: comment is redundant, that's just how recover() works.

Reply:
Yeah, but that assumes you are familiar with `recover`. We don't really use `recover` in other places.